computeGlobalWeightedIntegral.py

```diff
@@ -16,7 +16,7 @@ def computeGlobalWeightedIntegral(NEL, varCon, varF, areas, jacobians, SpectralE
        # Loop over each element and compute the sum
        INT = 0.0
        if not SpectralElement:
-              INT = np.dot(areas,varF)
+              INT = varF.dot(areas)
```

Comment:
Is this the bug ?! Doh. If it is, I'm going to shoot myself.

Aren't the shapes of areas and varF the same ? I need to debug this to see why it would make a difference.

Reply:
From https://docs.scipy.org/doc/numpy/reference/generated/numpy.dot.html:

```
numpy.dot(a, b, out=None)
Dot product of two arrays. Specifically,

If both a and b are 1-D arrays, it is inner product of vectors (without complex conjugation).
```


This should be the same as varF.dot(areas), no?

Reply:
No this isn't the bug. That code is identical.

Reply:
Actually, (and this is annoying) I have yet to actually find a proper bug here... I just did an independent check to the routines, and reverted back to a previous simpler definition of the norms. Otherwise, I didn't really make any functional changes to any of the routines.